sharepoint/sharepoint-ps/sharepoint-online/Get-SPODataAccessGovernanceInsight.md

```diff
@@ -0,0 +1,130 @@
+---
+external help file: Microsoft.Online.SharePoint.PowerShell.dll-Help.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version:
+schema: 2.0.0
+author: pvrk
+ms.author: pvrk
+manager: 
+ms.reviewer:
+---
+
+# Get-SPODataAccessGovernanceInsight
+
+## SYNOPSIS
+
+Lists various reports available in 'Data Access Governance' module (DAG) in SharePoint Admin Center.
+
+## SYNTAX
+
+### GetAllReportsParameterSet
+```
+Get-SPODataAccessGovernanceInsight -ReportEntity <ReportEntityEnum> [-WorkLoad <WorkloadEnum>]
+ [-ReportType <ReportTypeEnum>] [<CommonParameters>]
+```
+
+### GetReportParameterSet
+```
+Get-SPODataAccessGovernanceInsight -ReportID <Guid> [<CommonParameters>]
+```
+
+## DESCRIPTION
+
+Fetch details of various reports available in 'Data Access Governance' module (DAG) in SharePoint Admin Center
+
+## EXAMPLES
+
+### Example 1
+
+```powershell
+Get-SPODataAccessGovernanceInsight -ReportEntity EveryoneExceptExternalUsersForItems
+```
+
+The above command fetches all DAG reports about 'Everyone except external users' attached to a item i.e., file/folder/list. The output consists of important parameters such as Status, ReportID, number of sites in the report and other user provided values during report generation.
+
+## PARAMETERS
+
+### -ReportEntity
+
+Specifies the 'potential oversharing' scenario as captured by DAG report given during the report creation.
```

Comment:
>Specifies the 'potential oversharing' [](http://example.com/codeflow?start=0&length=37)

This sentence could be clarified: what does "ReportEntity" have to do with "potential oversharing"?